styleguide/derek/global-components/footer/_footer.scss

```diff
@@ -46,6 +47,7 @@
 .footer-tab-link {
   color: $white;
   font-size: 1.3rem;
+  font-weight: 500;
```

Comment:
We only have the 500 weight installed on the rackspace_spa theme, so this change won't make any sort of difference elsewhere on the site.